xml/System.Security.Authentication/SslProtocols.xml

```diff
@@ -30,11 +30,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Examples  
- The following code example demonstrates creating a <xref:System.Net.Sockets.TcpClient> that uses the <xref:System.Net.Security.SslStream> class to communicate with a server.  
-  
- [!code-cpp[NclSslClientSync#0](~/samples/snippets/cpp/VS_Snippets_Remoting/NclSslClientSync/CPP/clientsync.cpp#0)]
- [!code-csharp[NclSslClientSync#0](~/samples/snippets/csharp/VS_Snippets_Remoting/NclSslClientSync/CS/clientsync.cs#0)]  
-  
+ See <xref:System.Net.Security.SslStream> for code examples.  
```

Comment:
We don't usually add the section if you don't have examples. Can't you just use the same here?